src/libraries/System.Text.Json/src/System/Text/Json/Serialization/JsonSerializerOptions.Converters.cs

```diff
@@ -205,15 +206,16 @@ private JsonConverter GetConverterFromAttribute(JsonConverterAttribute converter
             }
             else
             {
-                ConstructorInfo ctor = type.GetConstructor(Type.EmptyTypes);
+                ConstructorInfo ctor = type.GetConstructor(Type.EmptyTypes)!;
```

Comment:
`type.GetConstructor` can return null and hence there should be a null check here to prevent a null reference exception.

See https://github.com/dotnet/runtime/issues/399

Reply:
hm, not sure why i banged here.